configapp/configapp.py

```diff
@@ -21,16 +23,60 @@ def index():
 
 @app.route('/live')
 def live():
-    return render_template('live.html')
+    data = {}
+    subdirs = [d for d in os.listdir(predict_dir)
+               if os.path.isdir(os.path.join(predict_dir, d))]
+
+    if not subdirs:
+        return render_template('live.html', nopredictions=True)
+
+    subdirs.sort()
```

Comment:
We assume that directory format is `yyyy-mm-dd`. The alphabetical sort is sufficient here.